tests/_phlop/run/test_test_cases.py

```
@@ -0,0 +1,27 @@
+#
+#
+
+import os
+import sys
+import unittest
+
+
+class PhlopGitTestCasesTest(unittest.TestCase):
+    def test_exec(self):
+        True
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)